presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/thrift/InMemoryThriftMetastore.java

```diff
@@ -283,6 +283,20 @@ public synchronized void alterTable(String databaseName, String tableName, Table
         return tables.build();
     }
 
+    @Override
+    public synchronized List<String> getTablesWithParameter(String databaseName, String parameterKey, String parameterValue)
+    {
```

Comment:
Validate `parameterKey`, `parameterValue` aren't null. Otherwise you can have NPE.